src/screen.rs

```diff
@@ -1114,22 +1118,53 @@ impl Screen {
             Cancel => {
                 self.error_file = None;
                 self.set_search(None);
+                self.clear_repeat_count();
```

Comment:
BTW, I've been wondering if we should make `Cancel` apply in priority order to things.  e.g. first `Esc` keypress clears the repeat count (if any), next clears the error message (if any), next clears the search (if any), next clears the error file (if any).  What do you think?